source/docker.rst

```diff
@@ -0,0 +1,81 @@
+Docker
+===================================
+
+.. toctree::
+   :maxdepth: 2
+   :hidden:
+
+*************
+Docker images
+*************
+
+All reusable Docker images should abide by the following rules:
+
+* Where less frequently used packages are desired, consider making a base image and use that for the generic packages.
+* Use the tagging structure, :code:`0.9` for images - this should be reflected in GitHub & Dockerhub releases
+
+*******
+Distro
+*******
+
+For build tools, where possible we should make use of Alpine Linux as this is a very lean image. Additional packages can be added with:
+
+.. code-block:: bash
+
+        apk add --update {packages}
+
+When mirroring server environments, the distro should always match the server. For Google App Engine we should use debian:wheezy.
+
+***********
+Dockerfile
+***********
+
+* All images should be ephemeral
+* MAINTAINER set to devops@rehabstudio.com
+* WORKDIR should point at the mounted root
+* COPY instead of ADD, unless you want to unpack an archive
+* RUN should be used efficiently. Multi-line when possible, e.g. 
+
+.. code-block:: bash
+
+        RUN apt-get install -y curl \
+	wget \
+	perl
+
+* CMD should be in the format :code:`CMD [“exec”, “param1”, “param2”]`, not :code:`CMD exec param1 param2`
+* ENTRYPOINT don’t use this unless you have a specific need to
+* VOLUME to be used where data should be added to a volume on mount. The container shouldn’t rely on anything being mounted from host though
+* USER shouldn't be specified unless you're doing some user-specific actions inside the container. The default user (e.g. root) is fine for most containers.
+* ONBUILD shouldn’t be used unless there’s a really good reason to do so
+* Use a .dockerignore when it makes sense - exclude what doesn’t need added to the container
```

Comment:
I would add a note _specifically_ about potential secrets being added in the context, like `.env` files or passed in `docker-compose.yml` and the like, and that `.dockerignore` should be utilized to avoid this where possible.